src/components/Lookup/index.js

```diff
@@ -72,6 +75,9 @@ class Lookup extends Component {
             [DOWN_KEY]: this.handleKeyDownPressed,
             [ENTER_KEY]: this.handleKeyEnterPressed,
         };
+        this.handleScrollDownArrowHover = this.handleScrollDownArrowHover.bind(this);
+        this.handleScrollUpArrowHover = this.handleScrollUpArrowHover.bind(this);
+        this.stopArrowScoll = this.stopArrowScoll.bind(this);
```

Comment:
@LeandroTorresSicilia we should have util here that does all the bindings, we have exactly the same code every where in-class components, something like this

```js
bind({
    methods: [
        'handleScrollDownArrowHover',
       'handleScrollUpArrowHover',
    ],
    to: this,
});
```
it will be more semantic and reduce litte the component code, agree??

Reply:
Yes, I agree with you, I will create an issue for this